features/specs/gezag-meerderjarige/gezag-gedeeltelijke-bepaling.feature

```diff
@@ -0,0 +1,325 @@
+        | toelichting                      | Gezag is niet volledig te bepalen omdat de gegevens van een ouder niet worden bijgehouden. Mogelijk heeft deze ouder ook gezag.  |
+
+      Voorbeelden:
+      | burgerservicenummer  |
+      | 000000012            |
+      | 000000036            |
+
+    # huidige gezag niet te bepaling melding:  gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: ouder1 van bevraagde persoon is niet in BRP geregistreerd
+    Abstract Scenario: wanneer een ouder niet is ingeschreven in de BRP kan voor de andere ouder (ouder 2) alsnog gezag worden bepaald er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven de persoon 'Jorike' met burgerservicenummer '000000064'
+        * is meerderjarig
+        * is niet ingeschreven in de BRP
+      En persoon 'Marcella'
+        * heeft 'Jorike' als ouder 1 met erkenning bij geboorteaangifte
+        * heeft 'Leenderd' als ouder 2
+      Als gezag wordt gezocht met de volgende parameters
+        | naam                | waarde                |
+        | burgerservicenummer | <burgerservicenummer> |
+      Dan heeft de response een persoon met de volgende gegevens
+        | naam                | waarde                |
+        | burgerservicenummer | <burgerservicenummer> |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+        | naam                             | waarde                                                                                                                           |
+        | type                             | EenhoofdigOuderlijkGezag                                                                                                         |
+        | minderjarige.burgerservicenummer | 000000036                                                                                                                        |      
+        | ouder.burgerservicenummer        | 000000024                                                                                                                        |
+        | toelichting                      | Gezag is niet volledig te bepalen omdat de gegevens van een ouder niet worden bijgehouden. Mogelijk heeft deze ouder ook gezag.  |
+
+      Voorbeelden:
+      | burgerservicenummer  |
+      | 000000024            |
+      | 000000036            |
+ 
+    # huidige gezag niet te bepaling melding: Voor de bevraagde persoon moet niet_ouder geregistreerd staan in BRP
+    Abstract Scenario: wanneer een ouder (niet ouder) niet is ingeschreven in de BRP kan voor de andere ouder (ouder 1) alsnog gezag worden bepaald er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven de persoon 'Boris' met burgerservicenummer '000000096'
+        * is meerderjarig
+        * is niet ingeschreven in de BRP
+      En persoon 'Marcella'
+        * heeft 'Pieternella' als ouder 1 
+      En 'Pieternella' en 'Boris' zijn gehuwd
```

Comment:
onduidelijk uit de stap is wanneer dit huwelijk begon. Is de minderjarige staande huwelijk geboren, dan is er gewoon gezamenlijk gezag. We hebben van beide ouders een bsn, dus kan je dat gewoon leveren. Is het kind niet staande huwelijk geboren, dan heeft alleen moeder gezag. Die kan je dan ook gewoon leveren.

Een andere situatie - de situatie die ik bedoelde in #285 - is wanner een ouder niet in BRP en ook niet in RNI staat. De ouder heeft dan geen bsn en dat kunnen we nog niet leveren. Daarvoor gaan we nu gezagsrelaties toevoegen naast gezag, waarbij de ouder(s) in gezagsrelaties burgerservicenummer niet required is.